src/Apps/Artist/routes.tsx

```diff
@@ -135,6 +135,7 @@ export const routes: RouteConfig[] = [
           OverviewRoute.preload()
         },
         displayNavigationTabs: true,
+        ignoreScrollBehavior: true,
```

Comment:
@sweir27 - added better scroll management in this PR which fixed the artist tabs behavior -- now you can click them and don't get thrown back to the top of the page.